plugins/ci/cmd/insights-ci/main.go

```diff
@@ -34,128 +24,95 @@ func exitWithError(message string, err error) {
 }
 
 func main() {
-	ctx := context.Background()
-	const configFile = "./fairwinds-insights.yaml"
-	configHandler, err := os.Open(configFile)
+	cloneRepo := strings.ToLower(strings.TrimSpace(os.Getenv("CLONE_REPO"))) == "true"
```

Comment:
seems like maybe you could get this environment variable in `setupConfiguration` instead of having to pass it all the way through